src/main/webapp/WEB-INF/jsp/user-page.jsp

```diff
@@ -47,7 +47,10 @@
       </div>
       <br/>
       <div id="naviposts">
-        <button id="posts-button" onclick="switchTab('posts')">Posts</button> <button id="gallery-button" onclick="switchTab('gallery')">Gallery</button>
+        <label>
+          <p>Viewing Mode: </p>
+          <button id="posts-button" onclick="switchTab('posts')">All Posts</button> <button id="gallery-button" onclick="switchTab('gallery')">Gallery</button>
```

Comment:
I'm wondering if people will know what "Gallery" means. Maybe change it to something like "Image Gallery" or "Image Posts" or something?

Alternatively, maybe add some instructions to the homepage? Something like: "Try uploading images and then clicking the Gallery button to see all your image posts".

I don't feel super strongly, but I think it's a good idea to make things obvious for people completely new to our project.

Reply:
Oh whoops I just saw that Gloria left a similar comment. Didn't mean to pile on, sorry!